### PR TITLE
add name param to ctc_ops

### PR DESCRIPTION
**Pull request by afpro**

### CODE REVIEW

File: tensorflow/python/ops/ctc_ops.py [hunk 1]
```diff
@@ -31,7 +31,8 @@
 def ctc_loss(labels, inputs, sequence_length,
              preprocess_collapse_repeated=False,
              ctc_merge_repeated=True,
-             ignore_longer_outputs_than_inputs=False, time_major=True):
+             ignore_longer_outputs_than_inputs=False, time_major=True,
+             name=None):
```
Comment (caisq) [inline tensorflow/python/ops/ctc_ops.py]: The added `name` kwarg needs to be documented in the Args section of the doc string below.

File: tensorflow/python/ops/ctc_ops.py [hunk 2]
```diff
@@ -149,7 +150,8 @@ def ctc_loss(labels, inputs, sequence_length,
       sequence_length,
       preprocess_collapse_repeated=preprocess_collapse_repeated,
       ctc_merge_repeated=ctc_merge_repeated,
-      ignore_longer_outputs_than_inputs=ignore_longer_outputs_than_inputs)
+      ignore_longer_outputs_than_inputs=ignore_longer_outputs_than_inputs,
+      name=name)
```
Comment (caisq) [inline tensorflow/python/ops/ctc_ops.py]: Can you add a unit test to test that using `ctc_loss` with the new `name` kwarg works.

Comment (afpro) [reply tensorflow/python/ops/ctc_ops.py]: ok, i will do this as soon as possible

Comment (afpro) [reply tensorflow/python/ops/ctc_ops.py]: done

**Comment by drpngx** [comment]
Jenkins, test this please.

@caisq @ebrevdo how does the new version look?

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
Once we like this change, we should trigger an API review.

**Comment by sb2nov** [comment]
@afpro can you take a look at the failing tests. so that we can move the review forward

**Comment by afpro** [comment]
@sb2nov i'll check this after work, cause i'm using a mac for work, linux at home.

**Comment by ebrevdo** [comment]
Looks like the API has now changed; so LGTM but we'll need API approvers to describe how you should update the API metadata.

**Comment by ebrevdo** [comment]
@martinwicke ptal - do we need to run the api regeneration?

**Comment by martinwicke** [comment]
Yes, the instructions are with the failed test.

**Comment by martinwicke** [comment]
@afpro, could you run the API golden regeneration (must be on Linux, py2), and add the changed files to this PR?

**Comment by afpro** [comment]
'name' param is not necessary, close this PR.

**Pull request closed without merging**